engines/postgres.py

```diff
@@ -96,8 +96,6 @@ def insert_data_from_file(self, filename):
     def insert_statement(self, values):
         """Returns a SQL statement to insert a set of values"""
         statement = Engine.insert_statement(self, values)
-        if isinstance(statement, basestring):
-            statement = statement.decode("utf-8", "ignore")
```

Comment:
Is the need for these lines removed by the new default encoding? (to be honest I'm not sure why they are there, but just wanted to make sure we weren't breaking an edgecase somewhere)